userguide/virtualmachines.rst

```diff
@@ -174,6 +174,10 @@ a Virtual Machine (VM) type are described in
    | 1        | Enable VNC         | checkbox       | Add a VNC remote connection. Requires *UEFI* booting.                                         |
    |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
+   | 1        | Delay VM Boot      | checkbox       | Wait to start VM until VNC client connects. Only appears when :guilabel:`Enable VNC` is set.  |
+   |          | until VNC Connects |                |                                                                                               |
```

Comment:
```suggestion
   |          | Until VNC Connects |                |                                                                                               |
```

Reply:
It is lowercase in the web interface.

Reply:
Just fixed that: https://github.com/freenas/webui/pull/2650